apps/core/management/commands/create_intensive_test_data.py

```
@@ -0,0 +1,310 @@
+    ("Cleaning supplies", "household", 2, [0, 1, 2], Decimal("28.60")),
+    ("Birthday dinner", "celebrations", 3, [0, 1, 2, 3], Decimal("312.00")),
+    ("Hiking gear rental", "activities", 0, [0, 1, 2], Decimal("66.00")),
+]
+
+
+class Command(BaseCommand):
+    """Command for creating an intensive set of test data"""
+
+    help = "Creates an intensive set of test data (same users as restore_test_data, many more rooms/transactions/debts)"
+
+    def handle(self, *args, **options):
```

Ruff (0.15.12)

[warning] Missing type annotation for `*args`

(ANN002)